app/i18n/locales/en.js

```diff
@@ -264,6 +264,7 @@ export default {
 	No_messages_yet: 'No messages yet',
 	No_Reactions: 'No Reactions',
 	No_Read_Receipts: 'No Read Receipts',
+	no_search_result_found: 'No search results found',
```

Comment:
You can use `No_results_found` and not need create it.

Reply:
Hi @djorkaeffalexandre , I am sorry, I didn't get you.